cat-int.Rmd

```diff
@@ -296,8 +296,8 @@ How do we fit this model in `R`? There are a number of ways.
 One method would be to simply create a new variable, then fit a model like any other.
 
 ```{r, eval = FALSE}
-autompg$x3 = autompg$disp * autompg$domestic # THIS CODE NOT RUN!
-do_not_do_this = lm(mpg ~ disp + domestic + x3, data = autompg) # THIS CODE NOT RUN!
+autompg$x3 = autompg$disp * autompg$domestic # THIS CODE DOES NOT RUN!
```

Comment:
How about instead of `does`, we use `is`?

```suggestion
autompg$x3 = autompg$disp * autompg$domestic # THIS CODE IS NOT RUN!
do_not_do_this = lm(mpg ~ disp + domestic + x3, data = autompg) # THIS CODE IS NOT RUN!
```